docs/source/whatsnew/0.8.4.txt

```diff
@@ -38,10 +43,12 @@ Bug Fixes
 Performance
 ~~~~~~~~~~~
 
-* Speeds up `AssetFinder.lookup_symbol` by adding an extension, 
-`AssetFinderCachedEquities`, that loads equities into dictionaries and
-then directs `lookup_symbol` to these dictionaries to find matching equities
- (:issue:`830`).
+.. py:currentmodule:: zipline.assets.assets
```

Comment:
just use the full path, this is stateful and someone later in the file might not know to reset this.